client/src/components/Button.js

```diff
@@ -0,0 +1,136 @@
+import styled from "styled-components";
+
+export default function MonthButton({ filterAll, resetAllFilters, month }) {
+  return (
+    <WrapButton>
+      <Button
```

Comment:
It might be shorter and maybe also more comfortable to create a new component Button so that you don't have to include it manually 12 times. It should be possible to declare an array (e.g. const months = ("Januar", "Februar", etc.) you can map over and give out a button for each month.